after/ftplugin/qf.vim

```diff
@@ -78,7 +78,7 @@ nnoremap <silent> <buffer> <plug>(quickr_preview) :call QFList(line("."))<CR>
 " Default key maps {{
 if g:quickr_preview_keymaps
     nmap <leader><space> <plug>(quickr_preview)
-    nmap <space> <plug>(quickr_preview)
+    nmap <buffer> <space> <plug>(quickr_preview)
```

Comment:
I'd rather not have 2 default mappings. Users can set this in their own config if they want. Rest of the change looks fine to me.